ga4gh/backend.py

```diff
@@ -822,3 +844,102 @@ def __init__(self, dataDir):
         for datasetDir in datasetDirs:
             dataset = datasets.FileSystemDataset(datasetDir, self)
             self.addDataset(dataset)
+
+        self._g2p_backend = GenotypePhenotypeBackend(
```

Comment:
Change this to ``self._g2pDataset = genotype_phenotype.G2PDataset(sources)``. There is only on backend object, and it gathers together all the disparate data sources.